core/src/main/java/com/linecorp/armeria/client/RefreshingAddressResolverGroup.java

```diff
@@ -90,7 +90,18 @@
         return builder.build();
     }
 
-    private final ConcurrentMap<String, CompletableFuture<CacheEntry>> cache = new ConcurrentHashMap<>();
+    private static Cache<String, CompletableFuture<CacheEntry>> buildDnsCache(String spec) {
```

Comment:
Could you move this private static method at the bottom of this class?
The reason that `dnsRecordTypes` is located at the top is that the method is used in the static initializer.